### DISCUSSION TITLE
nuclei problrm

### DISCUSSION THREAD

**Discussion by procry10** [projectdiscovery/nuclei#5801] [General]
Cloning into 'nuclei'...
remote: Enumerating objects: 44249, done.
remote: Counting objects: 100% (1852/1852), done.
remote: Compressing objects: 100% (1014/1014), done.
error: RPC failed; curl 56 GnuTLS recv error (-9): Error decoding the received TLS packet.
error: 2186 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

pls any solition

**Comment by GeorginaReeder** [comment]
Hi there @procry10 !

As a first step, could you let us know what command you used to get this output? If you have any screenshots, that would also be very helpful. We can then take a closer look!
